docs/en/connector-v2/sink/Redis.md

```diff
@@ -12,21 +12,24 @@ Used to write data to Redis.
 
 ## Options
 
-|      name      |  type  |       required        | default value |
-|----------------|--------|-----------------------|---------------|
-| host           | string | yes                   | -             |
-| port           | int    | yes                   | -             |
-| key            | string | yes                   | -             |
-| data_type      | string | yes                   | -             |
-| batch_size     | int    | no                    | 10            |
-| user           | string | no                    | -             |
-| auth           | string | no                    | -             |
-| db_num         | int    | no                    | 0             |
-| mode           | string | no                    | single        |
-| nodes          | list   | yes when mode=cluster | -             |
-| format         | string | no                    | json          |
-| expire         | long   | no                    | -1            |
-| common-options |        | no                    | -             |
+|       name       |  type  |       required        | default value |
+|------------------|--------|-----------------------|---------------|
+| host             | string | yes                   | -             |
+| port             | int    | yes                   | -             |
+| key              | string | yes                   | -             |
+| data_type        | string | yes                   | -             |
+| batch_size       | int    | no                    | 10            |
+| user             | string | no                    | -             |
+| auth             | string | no                    | -             |
+| db_num           | int    | no                    | 0             |
+| mode             | string | no                    | single        |
+| nodes            | list   | yes when mode=cluster | -             |
+| format           | string | no                    | json          |
+| expire           | long   | no                    | -1            |
+| value_field      | string | no                    | -             |
```

Comment:
The OptionRule of RedisSinkFactory should be consistent with the description in the documentation.